internal/app/server/server.go

```diff
@@ -0,0 +1,22 @@
+package server
+
+import (
+	"github.com/go-chi/chi/v5"
+	"github.com/kripsy/shortener/internal/app/handlers"
+)
+
+type MyServer struct {
+	Router  *chi.Mux
+	URLRepo string
+}
+
+func InitServer(URLRepo string, repo handlers.Repository) *MyServer {
+	m := MyServer{}
+	m.URLRepo = URLRepo
+
+	m.Router = chi.NewRouter()
+	m.Router.Post("/", handlers.SaveURLHandler(repo, m.URLRepo))
```

Comment:
Nit: you can use here an interface

Reply:
now i don't need interface, so SaveURLHandler is a method of a struct and work with field struct.